process/process.md

```diff
@@ -0,0 +1,291 @@
+# Preface
+
+This document defines processes adopted by Xen-Troops project. It
+should be followed by all Xen-Troops members. Parts of the document
+that describe commits, pull requests and such should be followed by
+external contributors as well.
+
+# Definition of Done
+
+Every task is considered completed when resulting changes are fully
+integrated into the codebase. This means that you have done your task
+only **after** your Pull Request(s) was/were merged. Before that
+moment your task is **not** done.
+
```

Comment:
Sometimes DoD for us means code is in upstream. I suggest having a section on how to upstream (including mess with using private emails but submitting on behalf work emails)